css/sitesummary.css

```diff
@@ -11,3 +11,25 @@
 .gridfield-button-link {
     margin-bottom: 12px;
 }
+
+.cms table.ss-gridfield-table tr td.col-Summary {
+    padding: 0;
+}
+
+a.package-summary__anchor {
+    color: inherit;
+    text-decoration: inherit;
+    display: block;
+    padding: 8px;
+}
+
+a.package-summary__anchor:hover {
+    color: inherit;
+    text-decoration: inherit;
+}
+
+a.package-summary__anchor:active {
```

Comment:
It might be worth combining these for now - `.package-summary__anchor:hover, .package-summary__anchor:active {...}`

Reply:
👍